stl/inc/functional

```diff
@@ -1209,6 +1215,170 @@ _NODISCARD bool operator!=(nullptr_t, const function<_Fty>& _Other) noexcept {
 }
 #endif // !_HAS_CXX20
 
+#if _HAS_CXX23
+template <class _Rx, class... Args>
+struct _Move_only_function_impl {
+    using _Invoke_t  = _Rx(__stdcall*)(uintptr_t* _Self, Args...);
+    using _Move_t    = void(__stdcall*)(uintptr_t* _Self, uintptr_t* _Src) _NOEXCEPT_FNPTR;
+    using _Destroy_t = void(__stdcall*)(uintptr_t* _Self) _NOEXCEPT_FNPTR;
+
+    _Invoke_t _Invoke;
+    _Move_t _Move;
+    _Destroy_t _Destroy;
+};
+
+template <class _Rx, class... Args>
+[[noreturn]] _Rx __stdcall _Function_throw_bad_call(uintptr_t*, Args...) {
+    _Xbad_function_call();
+}
+
+template <class _Rx, class... Args>
+inline constexpr _Move_only_function_impl<_Rx, Args...> _Null_move_only_function = {
+    _Function_throw_bad_call<_Rx, Args...>,

```

Comment:
`move_only_function::operator()` isn't required to throw `bad_function_call` (and `move_only_function<int() noexcept>::operator()` can't actually throw any exception). Consider just using `nullptr` here.

Reply:
Right, this is _Preconditions_. I would prefer `abort()` then - it will be still easier to debug this way.